CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contributing to fullstory-babel-plugin-annotate-react
+
+Thanks for taking the time to contribute!
+
+Here are some important resources to consider:
+
+  * [FullStory's Annotate React plugin for Web & Native](https://help.fullstory.com/hc/en-us/articles/360049493054-FullStory-s-Annotate-React-plugin-for-Web-Native)
+  * [Getting Started with FullStory React Native Capture](https://help.fullstory.com/hc/en-us/articles/360052419133)
+  * [README.md](./README.md)
+  * [FullStory Help Center](https://www.fullstory.com/)
+  * [FullStory Community](https://community.fullstory.com/)
+  * [FullStory Developer Guide](https://developer.fullstory.com/)
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help.
+
+## Table of Contents
+
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+    - [Making Code Changes](#making-code-changes)
+    - [Reporting Bugs](#reporting-bugs)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+- [Pull Request Process](#pull-request-process)
+
+## I Have a Question
+
+> Search for similar questions in our [Community Portal](https://community.fullstory.com/) and browse through our [Help Center](https://help.fullstory.com/hc/en-us) before asking a question.
```

Comment:
Should we link to the non `en-us` specific help center URL?
```suggestion
> Search for similar questions in our [Community Portal](https://help.fullstory.com/) and browse through our [Help Center](https://help.fullstory.com/hc/en-us) before asking a question.
```

Reply:
yes, but we want to keep the community center link it was suggested in ask-cx-support

Reply:
i also learned you can link to non-english articles like this:
```
# english
https://help.fullstory.com/hc/en-us/articles/360052419133

# without language specified
https://help.fullstory.com/hc/articles/360052419133
```